TbsCore/Core/WebBrowserInfo.cs

```diff
@@ -91,7 +104,7 @@ private void SetupChromeDriver(Access.Access access, string username, string ser
             options.AddArguments("--mute-audio");
 
             // Make browser headless to preserve memory resources
-            if (acc.Settings.HeadlessMode) options.AddArguments("headless");
+            // if (acc.Settings.HeadlessMode) options.AddArguments("headless");

```

Comment:
add in the release notes that headless doesn't work anymore

Reply:
sure